cfavml/src/danger/impl_avx512.rs

```diff
@@ -247,6 +304,62 @@ impl SimdRegister<f64> for Avx512 {
     }
 }
 
+impl Hypot<f64> for Avx512 {
+    #[inline(always)]
+    unsafe fn hypot(x: Self::Register, y: Self::Register) -> Self::Register {
+        // convert the inputs to absolute values
+        let (x, y) = (
+            _mm512_andnot_pd(_mm512_set1_pd(-0.0), x),
+            _mm512_andnot_pd(_mm512_set1_pd(-0.0), y),
+        );
+        // find the max and min of the two inputs
+        let (hi, lo) = (
+            <Avx512 as SimdRegister<f64>>::max(x, y),
+            <Avx512 as SimdRegister<f64>>::min(x, y),
+        );
```

Comment:
Same vibe as mentioned earlier, best to have these just as two separate lines rather than a tuple.